test/test_internals/deps_synapse.go

```diff
@@ -113,7 +114,7 @@ func MakeSynapse(domainName string, depNet *NetworkDep, signingKeyFilePath strin
 	}
 	synContainer, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
 		ContainerRequest: testcontainers.ContainerRequest{
-			Image:        "ghcr.io/element-hq/synapse:v1.110.0",
+			Image:        "ghcr.io/element-hq/synapse:v1.116.0",
```

Comment:
upgrading this should be a separate PR please

Reply:
https://github.com/t2bot/matrix-media-repo/pull/627 -- I'll rebase this PR onto that one when/if it gets merged.  Until then, I'll mark this PR as a draft to prevent merge issues.